Makefile.amigaos

```diff
@@ -25,25 +31,27 @@ LINK		= vc
 CC_PPC		= vc
 LINK_PPC	= vc
 
+CC_VERS= [VBCC]
+
 INCLUDES_COMMON	= -IDevkits:sdk/classic/MCC_Guigfx/Developer/C/Include -IDevkits:sdk/classic/MCC_Texteditor/Developer/C/Include
 INCLUDES	= -IDevkits:sdk/classic/ndk_39/include/include_h -IDevkits:sdk/classic/MUI/Developer/C/Include $(INCLUDES_COMMON)
-INCLUDES_OS4= -ISDK:include/include_h -ISDK:MUI/C/include $(INCLUDES_COMMON)
+INCLUDES_OS4= -ISDK:include/include_h -ISDK:Local/common/include $(INCLUDES_COMMON)
```

Comment:
SDK has MUI under its folder and not under Local/common. I am not sure we need to change this.

Reply:
OK. I have my MUI sdk inside local, will change it to path you describe.